.specify/memory/constitution.md

```diff
@@ -0,0 +1,139 @@
+<!--
+Sync Impact Report:
+Version Change: N/A → 1.0.0
```

Comment:
our version would be 0.0.1 if anything. Though I think it's even a little premature for that.